lib/created_message_queue_handler.ts

```diff
@@ -101,6 +103,33 @@ const messageContainerName = getRequiredStringEnv("MESSAGE_CONTAINER_NAME");
 const storageConnectionString = getRequiredStringEnv("QueueStorageConnection");
 const queueConnectionString = getRequiredStringEnv("QueueStorageConnection");
 
+const documentClient = new DocumentDBClient(cosmosDbUri, {
+  masterKey: cosmosDbKey
+});
+
+const messageStatusModel = new MessageStatusModel(
+  documentClient,
+  messageStatusCollectionUrl
+);
+
+// needed for retries
```

Comment:
"We cannot use Functions bindings for retries because ... so we have to instantiate a client for the Queue storage"